sql/scriptdev2/scriptdev2.sql

```diff
@@ -2038,7 +2038,7 @@ INSERT INTO script_texts (entry,content_default,sound,type,language,emote,commen
 (-1409004,'The runes of warding have been destroyed! Hunt down the infedels my bretheren.',8039,6,0,0,'majordomo SAY_SPAWN'),
 (-1409005,'Ashes to Ashes!',8037,1,0,0,'majordomo SAY_SLAY'),
 (-1409006,'Burn mortals! Burn for this transgression!',8036,1,0,0,'majordomo SAY_SPECIAL'),
-(-1409007,'Impossible! Stay your attack mortals! I submitt! I submitt!',8038,1,0,0,'majordomo SAY_DEFEAT_1'),
+(-1409007,'Impossible! Stay your attack, mortals... I submit! I submit! Brashly, you have come to wrest the secrets of the Living Flame! You will soon regret the recklessness of your quest. I go now to summon the lord whose house this is. Should you seek an audience with him, your paltry lives will surely be forfeit! Nevertheless, seek out his lair, if you dare!',8038,1,0,0,'majordomo SAY_DEFEAT_1'),
```

Comment:
So it's more a case of missing the last string altogether which should then contain what has been added to the second string here? - or is there a 1409008 ?